internal/install/execution/platform_link_generator_integration_test.go

```diff
@@ -43,5 +43,15 @@ func TestGenerateShortNewRelicURL_InvalidAPIKey(t *testing.T) {
 	result, err := g.generateShortNewRelicURL(longURL)
 
 	require.Error(t, err)
-	require.Equal(t, len(result), len(longURL))
+	require.Equal(t, result, longURL)
+}
+
+func TestGenerateShortNewRelicURL_NonNewRelicURL(t *testing.T) {
+	t.Parallel()
+
+	g := NewPlatformLinkGenerator()
+	longURL := "https://www.youtube.com/watch?v=dQw4w9WgXcQ&ab_channel=RickAstley"
```

Comment:
nit: remove `&ab_channel=RickAstley` from the URL so as not to give away the punchline